hipify-clang/.SRCINFO

```diff
@@ -1,14 +1,14 @@
 pkgbase = hipify-clang
 	pkgdesc = Convert CUDA to Portable C++ Code
-	pkgver = 5.0.2
+	pkgver = 5.1.1
 	pkgrel = 1
 	url = https://github.com/ROCm-Developer-Tools/HIPIFY
 	arch = x86_64
 	license = unknown
```

Comment:
With ROCm 5 hipify-clang is released with MIT license. https://github.com/ROCm-Developer-Tools/HIPIFY/tree/roc-5.1.x

Reply:
Could you please also add a line to the package() function to install the license file?

Reply:
Sure!